contracts/EasyAuction.sol

```diff
@@ -98,7 +98,7 @@ contract EasyAuction is Ownable {
         uint96 volumeClearingPriceOrder;
         uint256 feeNumerator;
     }
-    mapping(uint256 => IterableOrderedOrderSet.Data) public sellOrders;
+    mapping(uint256 => IterableOrderedOrderSet.Data) internal sellOrders;
```

Comment:
This change is needed, as the compiler complains. Somehow it can not be public if we have two maps. This seems just a limitation from solidity. But it also not needed to be public

Reply:
I believe the issue is that the compiler is not able to create a viewer for `Data`. It's defined as:
```
struct Data {
    mapping(bytes32 => bytes32) nextMap;
    mapping(bytes32 => bytes32) prevMap;
}
```
The compiler doesn't know how to return the full mappings, and doesn't want to add two extra input values that are to be passed to the struct to get a printable type. I think it's ok to make it internal and add two independent view method for `nextMap` and `prevMap`.